troubleshooting/extra-android-permissions-always-set.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Extra Android permissions always set
+description: The APK built by the cloud service has more required permissions than I set in my project settings.
+type: troubleshooting
+page_title: Unwanted permissions on Android build
+slug: extra-android-permissions-always-set
+position:
+tags: appbuilder, android-permissions
+teampulseid:
+ticketid: 838093, 1082473
+pitsid:
+
+---
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.ab-s}} ({{site.ab}})</td>
+  <tr>
+    <td>Mobile development type</td>
+    <td>{{site.ac}} app</td>
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>
+  </tr>
+</table>
+
+## Description
+When I build my application the app asks for permissions that I didn’t set up. Permissions are added although not included in any way in the `config.xml` or the `AndroidManifest.xml`.

```

Comment:
Build is a bit out of place here. I would go for something like this:
**When I install my application on a device, it asks for...**